src/ConfigEditor.tsx

```diff
@@ -54,26 +99,78 @@ export class ConfigEditor extends PureComponent<Props, State> {
         <div className="gf-form">
           <FormField
             label="Servers"
+            labelWidth={8}
             onChange={this.onBootstrapServersChange}
             value={jsonData.bootstrapServers || ''}
             placeholder="broker1:9092, broker2:9092"
+            inputWidth={30}
```

Comment:
The broker string for 3 AWS MSK brokers in is 190 characters.  The default field width was very painful to look for typos in.